src/main/java/net/dries007/tfc/objects/blocks/wood/BlockSupport.java

```diff
@@ -53,6 +54,11 @@
     private static final AxisAlignedBB CONNECTION_E_AABB = new AxisAlignedBB(0.6875D, 0.625D, 0.3125D, 1.0D, 1.0D, 0.6875D);
     private static final AxisAlignedBB CONNECTION_W_AABB = new AxisAlignedBB(0.0D, 0.625D, 0.3125D, 0.3125D, 1.0D, 0.6875D);
 
+    private static final int sRangeHor = ConfigTFC.General.FALLABLE.supportBeamRangeHor;
+    private static final int sRangeVert = ConfigTFC.General.FALLABLE.supportBeamRangeVert;
+    private static final int sRangeHorNeg = sRangeHor * -1;
+    private static final int sRangeVertNeg = sRangeVert * -1;
```

Comment:
Remove these fields and instead just directly reference the config values. If these are here it's essentially forcing a MC restart to modify the config values, as they won't be changed beyond first config load.